src/components/prompt/prompt.jsx

```diff
@@ -58,9 +65,15 @@ const PromptComponent = props => (
                 <div>
                     {props.isStage ?
                         <div className={styles.infoMessage}>
-                            <FormattedMessage
-                                {...messages.availableToAllSpritesMessage}
-                            />
+                            {props.varType === SCALAR_VARIABLE_TYPE ? (
```

Comment:
It might be good to pass in a boolean prop for whether or not this is a list type instead of passing in the varType itself. That way, we don't have to import SCALAR_VARIABLE_TYPE from scratch-blocks at this level.